### DISCUSSION TITLE
Documentation: Mesh network guidance - Connectivity Hub

### DISCUSSION THREAD

**Discussion by MiguelElGallo** [Azure/data-management-zone#280] [Q&A]
### Documentation Issue

Hello
In the image from here: 
Here: https://github.com/Azure/data-management-zone/blob/main/docs/guidance/DataManagementAnalytics-NetworkArchitecture.md#1-meshed-network-architecture-recommended
You cannot see the "The Connectivity Hub" , which you can see in the first image.
Is this The Connectivity Hub part of the data management landing zone? Which file/module does deploy it?
Thanks!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Azure/data-management-zone/blob/main/CODE_OF_CONDUCT.md)

**Comment by marvinbuss** [comment]
The connectivity Hub is part of the core [Enterprise-Scale Landing Zones reference design](https://github.com/Azure/Enterprise-Scale).

The Data Management Zone is not shown in the second image, because we are looking at a scenario that only involves the Connectivity Hub and Data Landing Zone. The Management Zone is not involved in these data movement activities, which is why this is not shown here. Do you suggest do mention that in the text?
